packages/editor/src/lib/editor/managers/LicenseManager.test.ts

```diff
@@ -1,35 +1,48 @@
+import { generateKeyPair, generateLicenseKey } from '../../utils/licensing'
 import { LicenseManager } from './LicenseManager'
 
 describe('LicenseManager', () => {
-	const licenseManager = new LicenseManager()
-
-	it('Checks if a license key was provided', () => {
-		const result = licenseManager.getLicenseFromKey('')
+	it('Checks if a license key was provided', async () => {
+		const licenseManager = new LicenseManager()
+		const result = await licenseManager.getLicenseFromKey('')
 		expect(result).toMatchObject({ isLicenseValid: false, reason: 'no-key-provided' })
 	})
-	/* it('Validates the license key', () => {
-		const invalidLicenseKey = generateKey(
+	it('Validates the license key', async () => {
+		const keyPair = await generateKeyPair()
```

Comment:
possible to just do `await generateKeyPair()` in a `before`? that way you can just generate it once and forget about it